src/script/view_model/CallingViewModel.ts

```diff
@@ -208,7 +208,7 @@ export class CallingViewModel {
       toggleMute: (call: Call, muteState: boolean) => {
         this.callingRepository.muteCall(call.conversationId, muteState);
       },
-      toggleScreenshare: (call: Call): void => {
+      toggleScreenshare: (call: Call): Promise<void> | void => {
```

Comment:
Let's make this function `async` and make sure everyone awaits it.

Reply:
done in https://github.com/wireapp/wire-webapp/pull/9476/commits/86493550256841f593925a38402a95f5e3cc5a2b